ci-operator/step-registry/openshift-qe/installer/bm/day2/regulus/openshift-qe-installer-bm-day2-regulus-ref.yaml

```diff
@@ -0,0 +1,121 @@
+ref:
+  as: openshift-qe-installer-bm-day2-regulus
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  cli: latest
+  commands: openshift-qe-installer-bm-day2-regulus-commands.sh
+  credentials:
+    - namespace: test-credentials # was: ci this entry injects the custom credential
+      name: cluster-secrets-metal-perscale-cpt
+      mount_path: /bm
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+    - name: BAREMETAL               # begin vars for the target cluster
+      default: "true"
+      documentation: |-
+        Set to true if the job is for baremetal
+    - name: KUBECONFIG_PATH 
+      default: "~/mno/kubeconfig"
+      documentation: |-
+        Set kubeconfig path inside the bm, default to ~/mno/kubeconfig
+    - name: LAB_CLOUD   
+      default: "cloud31"
+      documentation: |-
+        target cluster 
+
+    - name: REG_BRANCH              # begin vars for Regulus repo
+      default: "cpt"
+      documentation: |-
+        Regulus CPT/Prow branch
+    - name: REG_LATEST
+      default: "false"
+      documentation: |-
+        Clone the latest Regulus version. When enabled, REG_BRANCH gets ignored.
+    - name: REG_PR
+      default: ""
+      documentation: |-
+        If defined, use a specific regulus PR. When defined, REG_LATEST and REG_BRANCH get ignored.
+
+
+    - name: REG_KNI_USER            # begin vars for lab.config
+      default: "root"
+      documentation: |-
+        target cluster bastion login
+    - name: REG_DP
+      default: "rds-cpt"
+      documentation: |-
+        Regulus run tag
+    - name: OCP_WORKER_0
+      default: ""
+      documentation: |-
+        target cluser worker
+    - name: OCP_WORKER_1
+      default: ""
+      documentation: |-
+        target cluser worker
+    - name: OCP_WORKER_2
+      default: ""
+      documentation: |-
+        target cluser worker
+    - name: BM_HOSTS
+      default: ""
+      documentation: |-
+        BM host for Ingress/Egress
+    - name: REG_SRIOV_NIC
```

Comment:
I think this variable should be stored on vault, because there we can have different values for different clouds. Happy to help you with this, ping me on slack, I can give you vault access. @HughNhan

Reply:
Once you have the variable (per cloud) stored in vault, you can load it as we do here: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh#L12

Reply:
I would like to evaluate the strategy of what to do with these vars again at a later time when our CPT setup is more mature. To decide these env's, we need a domain expert to analyze the worker nodes, and the current state of the system after the prior Steps in the chain have finished. For now,  we can rely on the regulus  "smart" config util to figure out dynamically what those values should be. This smart config always runs during the Regulus setup phase.

Reply:
If you want to evaluate the strategy later the more reason to put this in vault, you can change it at any time in point without opening PRs, seeking approvals, rehearsals,, etc.
You are the domain expert for data-plane testing, and you already are familiar with both allocations. There are only a couple of SRIOV enabled interfaces in each lab, this should very easy/quick to add to vault.
One of the SRIOV interfaces is already there (for both metal-perfscale-cpt and metal-perfscale-jetlag): **sriov_pf_name**, you could just add the second one.